cypress/integration/Email Change Tests/emailChange.specs.js

```diff
@@ -0,0 +1,42 @@
+ /// <reference types="cypress" />
+ import {emailTab} from '../../utils/changeEmailTab'
```

Comment:
```suggestion
 import { emailTab } from '../../utils/changeEmailTab';
 
```
Please fix the formatting wherever needed.